docs/40-troubleshooting.md

```diff
@@ -1,4 +1,47 @@
-# Getting logs without Azure Portal
+## Debugging and Troubleshooting
+
+### After creating my cluster, my nodes go to a 'startTaskFailed' state. Why?
+The most common case for this is that there was an issue with package installation or the custom script failed to run. To troubleshoot this you can simply download the output logs from the node.
+
+```r
+cluster <- makeCluster('myConfig.json')
+...
+# Get standard error file
+getClusterFile(cluster, "tvm-1170471534_2-20170829t072146z", "stderr.txt", downloadPath = "pool-errors.txt")
+
+# Get standard log file
+getClusterFile(cluster, "tvm-1170471534_2-20170829t072146z", "stderr.txt", downloadPath = "pool-errors.txt")
+```
+
+### My job never starts running. How can I troubleshoot this issue?
+This is often caused by the node not being in a good state. Take a look at the state of the nodes in the cluster to see if any of there are and nodes in an error or failed state. If not node is in a startTaskFailed state follow the instructions above. If the node is in an 'unknown' or 'unusable' state you may need to manually reboot the node.
```

Comment:
again to be super obvious - let's actually just tell them **how** to check the state of their nodes